### ISSUE TITLE
about the loss

### ISSUE THREAD

**Issue by jypython** [qqwweee/keras-yolo3#307]
why xy_loss. confidence_loss. class_loss use binary_crossentropy loss rather than  square loss, wh_loss use square loss,and it is weight is 0.5?why the weights of  localization error and  confidence_loss  of not contain bounding box are same

**Comment by jenskrauth** [comment]
I've got the same question. Can somebody explain from where the weight 0.5 in wh_loss is derived?

**Comment by S601327412** [comment]
> why xy_loss. confidence_loss. class_loss use binary_crossentropy loss rather than square loss, wh_loss use square loss,and it is weight is 0.5?why the weights of localization error and confidence_loss of not contain bounding box are same

hey gays，that is not weigth  MSE loss function formula is 1 / 2 (y' - y)^2, 1 / 2 just for derivative convenient。
